README.md

```diff
@@ -1 +1,75 @@
-# Private-Events
\ No newline at end of file
+<h3 align="center">Private Events - RoR</h3>
+
+<div align="center">
+
+[![Status](https://img.shields.io/badge/status-active-success.svg)](https://github.com/umairarshadbutt/Private-Events)
+[![Github Issues](https://img.shields.io/badge/GitHub-Issues-orange)](https://github.com/umairarshadbutt/Private-Events/issues)
+[![GitHub2 Pull Requests](https://img.shields.io/badge/GitHub-Pull%20Requests-blue)](https://github.com/umairarshadbutt/Private-Events/pulls)
+[![License](https://img.shields.io/badge/license-MIT-blue.svg)](/LICENSE)
+</div>
+<p align="center">The goal of this project is to understand and implementation of ActiveRecord’s associations </p>
+
+
+## Built With
+
+- Ruby on Rails
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- Text Editor (VSCode is suggested.)
+- Install [Ruby](https://ruby-doc.org/downloads/)
+- Install [Rails](https://guides.rubyonrails.org/getting_started.html)
+- Install [git](https://git-scm.com/downloads)
+
+### Setup
+
+- Open Terminal
+- Change directory to folder to download repository
+- Use `cd <file-path>`
+- Run `https://github.com/umairarshadbutt/Private-Events.git`
+- cd to the project `cd Private-Events`
+- Install gems `bundle install`
+- Install Yarn `yarn install --check-files` 
+- Run `rails server`
```

Comment:
## Add test instructions
- [ ] Once implemented, be sure to also add instructions to run the tests as well. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)).